client/src/test/scala/dwolla/cloudflare/AccessControlRuleClientSpec.scala

```diff
@@ -169,9 +170,53 @@ class AccessControlRuleClientSpec(implicit ee: ExecutionEnv) extends Specificati
       val http4sClient = fakeService.client(fakeService.deleteAccessRule(SampleResponses.Successes.removedAccessRule, accountId, accessRuleId1))
       val client = buildAccessControlRuleClient(http4sClient, authorization)
 
-      private val output = client.delete(accountId, accessRuleId1)
+      val output = client.delete(accountId, accessRuleId1)
+        .compile.toList.unsafeRunSync()
+
+      val expected = DeletedRule(
+        success = true,
+        errors = Some(List()),
+        messages = Some(List()),
+        id = Some(accessRuleId1),
+      )
+      output must contain(
+        expected
+      )
+    }
+
+    "delete access rule from account fails" in new Setup {
+      val http4sClient = fakeService.client(fakeService.deleteAccessRule(SampleResponses.Successes.removeAccessRuleFailed, accountId, accessRuleId1))
+      val client = buildAccessControlRuleClient(http4sClient, authorization)
+
+      val output = client.delete(accountId, accessRuleId1)
+        .compile.toList.unsafeRunSync()
```

Comment:
If the deletion fails, I'm surprised this succeeds and doesn't raise an exception (either an exception specific to a known failure type, or a generic "Cloudflare failed" kind of exception).